src/python_testing/TC_BOOLCFG_2_1.py

```diff
@@ -34,16 +34,16 @@ def desc_TC_BOOLCFG_2_1(self) -> str:
 
     def steps_TC_BOOLCFG_2_1(self) -> list[TestStep]:
         steps = [
-            TestStep(1, "Commissioning, already done", is_commissioning=True),
-            TestStep(2, "Read attribute list to determine supported attributes"),
-            TestStep(3, "Read SupportedSensitivityLevels attribute, if supported"),
-            TestStep(4, "Read CurrentSensitivityLevel attribute, if supported"),
-            TestStep(5, "Read DefaultSensitivityLevel attribute, if supported"),
-            TestStep(6, "Read AlarmsActive attribute, if supported"),
-            TestStep(7, "Read AlarmsSuppressed attribute, if supported"),
-            TestStep(8, "Read AlarmsEnabled attribute, if supported"),
-            TestStep(9, "Read AlarmsSupported attribute, if supported"),
-            TestStep(10, "Read SensorFault attribute, if supported"),
+            TestStep(1, "{comDutTH}.", "", is_commissioning=True),
```

Comment:
things I should have mentioned earlier - things get a bit weird if you pull these directly from the adoc files because all the stuff in the curly braces are adoc macros (er...they don't call them macros, but basically macros). 

So this will render correctly in the adoc (I think?), but it'll also show up directly like this in the test harness.

The solution Rene and I were kicking around was using actual python functions to return the text. ex like here: https://github.com/project-chip/connectedhomeip/pull/32182/files#diff-8bafa8e5179589abd80fe1365c30897596b32305372c4ee3986f1062b9afa0fd

The bonus there is you can consolidate all of this nicely by doing ex.
```
def constraints(min: typing.Union[int, str], max: stepnum: typing.Union[int, str], type: typing.Union[int, str, None):
  type_str = '' if type is none else f'dut_reply() a {type} value. '
  return f'{type_str}Value has to be between a range of {min} and {max}'

TestStep(3, read_attribute('SupporedSensitivitiyLevels'), constraints('uint8', 2, 10)) 
```

aside...man I'd love to get rid of these attribute constraint tests. Maybe in 1.5 we can start picking them off from the easy DM XMLs. This seems like SO much manual copy-paste for no reason.

Anyway @ReneJosefsen - what are your thoughts here? Do we like this style better than what's in the adoc currently? Is this going to be easier for people to keep in sync if we can generate the test plans automatically?

Reply:
From just an initial view, this seems like a move in the wrong direction. As you say, this does render fine in adoc once the import is there, but the macros/document attributes are not going to be adjusted accordingly here so it will look very weird when running this on TH.

So for this to work we need to have a similar text replacement in place first IMO.

Reply:
I updated `TC_BOOLCFG_2_2.py` to use macros.  See [here](https://github.com/project-chip/connectedhomeip/blob/f251be2951df7d6e856aa1f26c17c1ccaeac60d0/src/python_testing/TC_BOOLCFG_2_1.py#L38).

If we like this, I will update the rest of the `TC_BOOLCFG_*.py` files.